src/devices/cddadevice.cpp

```diff
@@ -42,17 +38,15 @@ CddaDevice::~CddaDevice() {}
 
 void CddaDevice::Init() {
   song_count_ = 0;  // Reset song count, in case it was already set
-  cdda_song_loader_.LoadSongs();
+  cdda_song_loader_.LoadSongs();  // will not perform costly operations if media
+                                  // has not changed since last time, but result
+                                  // in call to SongsLoaded again
 }
 
-void CddaDevice::Refresh() {
-  if (!cdda_song_loader_.HasChanged()) {
-    return;
-  }
-  Init();
-}
+void CddaDevice::Refresh() { Init(); }
```

Comment:
I know this is the existing pattern, but, in my option, it makes more sense to put the code in Refresh() and have Init() call Refresh().

Reply:
Done